src/components/environment-badge.tsx

```diff
@@ -22,13 +22,20 @@ export function EnvironmentBadge( { type, selected }: EnvironmentBadgeProps ) {
 			return 'bg-a8c-green-5 text-a8c-green-80';
 		}
 
+		if ( type === 'sandbox' ) {
+			return 'text-sandbox-text bg-sandbox-bg';
+		}
+
 		return '';
 	};
 
 	const getLabel = () => {
 		if ( type === 'staging' ) {
 			return __( 'Staging' );
 		}
+		if ( type === 'sandbox' ) {
+			return __( 'Sandbox' );
+		}
```

Comment:
These could be replaced with a dict object. Not a strong opinion, because it follows the same pattern as `getClassName`.

```tsx
	const labels: Record< EnvironmentType, string > = {
		staging: __( 'Staging' ),
		sandbox: __( 'Sandbox' ),
		production: __( 'Production' ),
	};
```
and then use it like:
```tsx
{ labels[ type ] }
```